spoolman.json

```
@@ -0,0 +1,42 @@
+{
+    "spoolman": {
+        "description": "Spoolman is a self-hosted Web service and database to manage your 3D printer filament spools.  <a href='https://github.com/Donkie/Spoolman' target='_blank'>https://github.com/Donkie/Spoolman</a>",
+        "version": "1.0",
+        "website": "https://github.com/Donkie/Spoolman",
+        "more_info": "This Rock-on uses the official Spoolman Docker image from Docker Hub, a muiltiarch image supporting amd64, arm64, and armhf architectures.</p>",
+        "ui": {
+            "https": false,
+            "slug": ""
+        },
+        "volume_add_support": false,
+        "containers": {
+            "spoolman": {
+                "image": "donkieyo/spoolman",
+                "tag":"edge",
+                "launch_order": 1,
+                "ports": {
+                    "8000": {
+                        "description": "Enter the port number you would like to use for the Spoolman Web interface.",
+                        "label": "Web interface port",
+                        "host_default": 7912,
+                        "protocol": "tcp",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/home/app/.local/share/spoolman": {
+                        "description": "Choose a share for Spoolman database storage",
+                        "label": "Spoolman Database Storage"
```

```suggestion
                        "label": "Spoolman Database Storage [e.g. spoolman-db]"
```
trying to move towards a somewhat standardized structure, and the suggestion has been to add a share name example  to the volume tags